.github/workflows/demo.yml

```diff
@@ -0,0 +1,41 @@
+name: Demo Project
+
+on:
+  push:
+  pull_request:
+    branches: [ main ]
```

Comment:
We don't have a main branch yet :)

Reply:
Hi, When we set branches to main it takes the default branch which is master, we can see Actions run here -> https://github.com/venky8951/puppeteer-sharp/actions